en/setup/dev_env_windows_cygwin.md

```diff
@@ -15,7 +15,7 @@ This topic explains how download and use the environment, and how it can be exte
 ## Installation Instructions {#installation}
 
 1. Download the latest version of the ready-to-use MSI installer from [Github](https://github.com/PX4/windows-toolchain/releases) or
-[S3](https://s3-us-west-2.amazonaws.com/px4-tools/PX4+Windows+Cygwin+Toolchain/PX4+Windows+Cygwin+Toolchain+0.8.msi)
+[S3](https://s3-us-west-2.amazonaws.com/px4-tools/PX4+Windows+Cygwin+Toolchain/PX4+Windows+Cygwin+Toolchain+0.9.msi)
```

Comment:
@MaEtUgR Is it possible to get a "latest" link that always maps to the latest version? That way I could avoid ever updating this doc just for new version?